### PR TITLE
translated 

### PR DESCRIPTION
**Pull request by spielers**
translated docs/adding-website-functionality.md

### CODE REVIEW

**Comment by spielers** [comment]
recipes.md translation is on its way. ill try to complete as soon as possible

File: docs/docs/add-404-page.md [hunk 1]
```diff
@@ -1,28 +1,19 @@
 ---
-title: Adding a 404 Page
+शीर्षक: 404 पेज जोड़ना
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `शीर्षक` => `title`

this should not be translated

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `जोड़ना` => `ऐड करना`

File: docs/docs/add-404-page.md [hunk 2]
```diff
@@ -1,28 +1,19 @@
...
 ---
 
-To create a 404 page create a page whose path matches the regex
-`^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`). Most often you'll want to create a React component page at
-`src/pages/404.js`.
+404 पृष्ठ बनाने के लिए एक पेज जिसका पथ regex से मेल खाता बनाने ^\/?404\/?$( /404/, /404, 404/or 404)। बहुधा आप एक प्रतिक्रिया घटक पृष्ठ बनाना चाहेंगे src/pages/404.js .
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पथ` => `पाथ`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `404 पृष्ठ बनाने के लिए एक पेज जिसका पथ regex से मेल खाता बनाने ^\/?404\/?$( /404/, /404, 404/or 404)। बहुधा आप एक प्रतिक्रिया घटक पृष्ठ बनाना चाहेंगे src/pages/404.js .` 
=> 
```
404 पेज बनाने के लिए एक पेज बनाये जिसका पाथ ^\/?404\/?$( /404/, /404, 404/or 404) regex से मेल खाता हो। ज़्यादातर आप एक React कौम्पोनॅन्ट पेज बनाना चाहेंगे `src/pages/404.js`।
```

Comment (arshadkazmi42) [reply docs/docs/add-404-page.md]: we are not translating React word. Refer [style-guide](https://github.com/gatsbyjs/gatsby-hi/blob/master/style-guide.md) for more info

File: docs/docs/add-404-page.md [hunk 3]
```diff
@@ -1,28 +1,19 @@
...
 
-Gatsby ensures that your 404 page is built as `404.html` as many static hosting
-platforms default to using this as your 404 error page. If you're hosting your
-site another way, you'll need to set up a custom rule to serve this file for 404
-errors.
+गैट्सबी यह सुनिश्चित करता है कि आपका 404 पेज उतना `404.html`ही स्टैटिक होस्टिंग प्लेटफॉर्म के रूप में बनाया गया है, जो इसे आपके 404 एरर पेज के रूप में उपयोग करने के लिए डिफ़ॉल्ट है। यदि आप अपनी साइट को दूसरे तरीके से होस्ट कर रहे हैं, तो आपको 404 त्रुटियों के लिए इस फ़ाइल को सेवा देने के लिए एक कस्टम नियम स्थापित करने की आवश्यकता होगी।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]:
```
गैट्सबी यह सुनिश्चित करता है कि आपका 404 पेज उतना `404.html`ही स्टैटिक होस्टिंग प्लेटफॉर्म के रूप में बनाया गया है, जो इसे आपके 404 एरर पेज के रूप में उपयोग करने के लिए डिफ़ॉल्ट है।
```
=>
```
Gatsby यह सुनिश्चित करता है कि आपका 404 `404.html` की तरह बना हो क्यूंकि ज़्यादातर स्टेटिक होस्टिंग प्लेटफॉर्म्स इसीको डिफ़ॉल्ट रूप से 404 एरर पेज की तरह इस्तेमाल करते है।
```

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `त्रुटियों` => `एर्ररस`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `सेवा देने` => `सर्व करने`

File: docs/docs/add-404-page.md [hunk 4]
```diff
@@ -1,28 +1,19 @@
...
 
-Because Gatsby creates this page for you by default, there is no need to configure
-it in your `gatsby-node.js` file.
+क्योंकि Gatsby डिफ़ॉल्ट रूप से आपके लिए यह पृष्ठ बनाता है, इसलिए इसे आपकी `gatsby-node.js` फ़ाइल में कॉन्फ़िगर करने की कोई आवश्यकता नहीं है ।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठ` => `पेज`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `नहीं है ।` => `नहीं है।`

File: docs/docs/add-404-page.md [hunk 5]
```diff
@@ -1,28 +1,19 @@
...
 
-When developing using `gatsby develop`, Gatsby uses a default 404 page that
-overrides your custom 404 page. However, you can still preview your 404 page by
-clicking "Preview custom 404 page" to verify that it's working as expected. This is
-useful when you're developing so that you can see all the available pages.
+प्रयोग करते समय `gatsby develop`, Gatsby एक डिफ़ॉल्ट 404 पृष्ठ का उपयोग करता है जो आपके कस्टम 404 पृष्ठ को ओवरराइड करता है। हालाँकि, आप अभी भी अपने 404 पृष्ठ का पूर्वावलोकन कर सकते हैं "कस्टम 404 पृष्ठ का पूर्वावलोकन करें" पर क्लिक करके यह सत्यापित करें कि यह अपेक्षित है। यह उपयोगी है जब आप विकसित कर रहे हैं ताकि आप सभी उपलब्ध पृष्ठों को देख सकें
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठ` => `पेज`

change this everywhere

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठों` => `पेजेज`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `को देख सकें` => `को देख सकें।`

File: docs/docs/add-404-page.md [hunk 6]
```diff
@@ -1,28 +1,19 @@
...
 
-The screenshot below shows the default 404 page that Gatsby creates.
-It also lists out all the pages on your website. Clicking the "Preview custom 404
-page" button will allow you to view the 404 page you created.
+नीचे दिया गया स्क्रीनशॉट Gatsby द्वारा बनाए गए डिफ़ॉल्ट 404 पृष्ठ को दिखाता है।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठ` => `पेज`

File: docs/docs/add-404-page.md [hunk 7]
```diff
@@ -1,28 +1,19 @@
...
+यह आपकी वेबसाइट के सभी पृष्ठों को भी सूचीबद्ध करता है। "पूर्वावलोकन कस्टम 404" पर क्लिक करना
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]:
```
यह आपकी वेबसाइट के सभी पृष्ठों को भी सूचीबद्ध करता है। "पूर्वावलोकन कस्टम 404" पर क्लिक करना 
पृष्ठ "बटन आपको आपके द्वारा बनाए गए 404 पृष्ठ को देखने की अनुमति देगा।
```
=>
```
ये वेब्सीटेस के सारे पेजेज को लिस्ट करता है। "प्रीव्यू कस्टम 404 पेज" बटन को क्लिक करने से ये आपको 404 पेज देखने की अनुमति देगा।
```

File: docs/docs/add-404-page.md [hunk 8]
```diff
@@ -1,28 +1,19 @@
...
+पृष्ठ "बटन आपको आपके द्वारा बनाए गए 404 पृष्ठ को देखने की अनुमति देगा।
 ![Gatsby Default 404 Page](images/gatsby-default-404.png)
 
-The screenshot below shows the custom 404 page.
+नीचे दिया गया स्क्रीनशॉट कस्टम 404 पृष्ठ दिखाता है।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठ` => `पेज`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.

i have added some review. Once these are fixed we will get this merged 👍

**Comment by spielers** [comment]
docs/docs/add-404-page.md  issues fixed please merge it

**Comment by spielers** [comment]
merge this page

**Comment by arshadkazmi42** [comment]
@spielers This pull request was supposed to be for `add-404-page`

I did review for only that. Also we follow 1 PR per page.

Can you send separate PRs for each page. Also, I will suggest not to create multiple pull requests. Create new pull requests once existing ones are merged.

Once you are done with separating the pages in different PRs. Ping here. I will review and get those merged one by one.

Let me know if you need any help

**Comment by spielers** [comment]
can you please tell me how can I create a separate pull request for each
page
sorry for the trouble but I am new here.

**Pull request closed without merging**